x/attribute/types/attribute.go

```diff
@@ -66,6 +67,9 @@ func (a Attribute) ValidateBasic() error {
 	if !isValidValueForType(a.AttributeType, a.Value) {
 		return fmt.Errorf("invalid attribute value for assigned type: %s", a.AttributeType)
 	}
+	if len(a.ConcreteType) > 200 {
```

Comment:
Just leaving a note here.

The longest msg-type URL is 80 characters long. So 200 seem like a reasonable max to me.